### PR TITLE
Extract SVGs to files

### PR DESCRIPTION
**Pull request by somebody1234**
### Pull Request Description
Extracts SVGs to its own module (name is tentative, currently `enso-assets` at `ide-desktop/lib/assets/`)
Closes https://github.com/enso-org/cloud-v2/issues/342

### Important Notes
At minimum the following scenarios should be tested:
- `ide watch`
- `ide build`
- `gui watch`
- `npm run watch-dashboard`
- `npm run build-dashboard`, then running a static http server, e.g. `python -m http.server`, in `lib/dashboard/build/`

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] ~~The documentation has been updated, if necessary.~~
- [x] ~~Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.~~
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] ~~Unit tests have been written where possible.~~
  - [x] ~~If GUI codebase was changed, the GUI was tested when built using `./run ide build`.~~

### CODE REVIEW

**Comment by somebody1234** [comment]
52 files changed :(
but only ~400 lines changed so it should be easy enough to code review

**Comment by somebody1234** [comment]
after finding (and reading) the original issue it seems the biggest hurdle is `electron-builder`, so definitely have to check with `ide build` first

**Comment by somebody1234** [comment]
so... somehow it works on `ide build`, reopening

File: app/ide-desktop/lib/assets/exclamation.svg
Comment (indiv0) [inline app/ide-desktop/lib/assets/exclamation.svg:1]: `exclamation.svg` looks empty in the GitHub SVG preview. Is this expected?

File: app/ide-desktop/lib/content/esbuild-config.ts [hunk 1]
```diff
@@ -106,7 +106,7 @@ export function bundlerOptions(args: Arguments) {
             '.css': 'copy',
             '.map': 'copy',
             '.wasm': 'copy',
-            '.svg': 'copy',
+            '.svg': 'file',
```
Comment (indiv0) [inline app/ide-desktop/lib/content/esbuild-config.ts:110]: What does this change do? Might want to document this.

File: app/ide-desktop/lib/dashboard/bundle.ts [hunk 2]
```diff
@@ -33,8 +33,9 @@ async function bundle() {
             path.resolve(THIS_PATH, 'src', 'index.html'),
             path.resolve(THIS_PATH, 'src', 'index.tsx')
         )
-        // eslint-disable-next-line @typescript-eslint/naming-convention
-        opts.loader = { '.html': 'copy' }
+        opts.loader = opts.loader ?? {}
+        opts.loader['.html'] = 'copy'
+        opts.loader['.svg'] = 'dataurl'
```
Comment (indiv0) [inline app/ide-desktop/lib/dashboard/bundle.ts]: What does this change do? Might want to document this.

File: app/ide-desktop/lib/dashboard/watch.ts [hunk 3]
```diff
@@ -29,8 +29,9 @@ OPTS.entryPoints.push(
 )
 OPTS.minify = false
 OPTS.write = false
-// eslint-disable-next-line @typescript-eslint/naming-convention
-OPTS.loader = { '.html': 'copy' }
+OPTS.loader = OPTS.loader ?? {}
+OPTS.loader['.html'] = 'copy'
+OPTS.loader['.svg'] = 'dataurl'
```
Comment (indiv0) [inline app/ide-desktop/lib/dashboard/watch.ts]: What does this change do? Should probably be documented.

**Review by indiv0 — APPROVED** [review]
LGTM, minus some questions regarding additional documentation. Only review for now, no QA.

File: app/ide-desktop/lib/assets/exclamation.svg
Comment (somebody1234) [reply app/ide-desktop/lib/assets/exclamation.svg:1]: yes - it is translucent white. it's a tiny bit clearer in chrome's preview
image

**Comment by somebody1234** [comment]
Changes should be documented - let me know if other lines were also supposed to be documented

**Comment by somebody1234** [comment]
Also note that (as noted in the new comments) the `dataurl` loader is not strictly necessary. If it's preferred to use the `file` loader for `watch-dashboard` and `build-dashboard` then it should be simple enough to switch

**Review by indiv0 — APPROVED** [review]
LGTM, thanks for the quick update!

**Review by wdanilo — APPROVED** [review]
Beautiful!

**Comment by PabloBuchu** [comment]
@somebody1234 can you merge it latest develop and try to run `ide build` prepended with compleate cleanup `./run git-clean`,remote target directory and `.enso-ci` cache?

**Comment by somebody1234** [comment]
umm i also just fixed the google icon being the github icon, i'm a little surprised nobody noticed (it appears it's been the github icon for the whole time it's existed)

**Comment by PabloBuchu** [comment]
QA 🟢 we should be ready to merge

**Review by PabloBuchu — APPROVED** [review]

**Pull request merged**